README.md

```diff
@@ -12,3 +12,59 @@ Almond is part of Open Thing Platform, a research project led by
 prof. Monica Lam, from Stanford University.  You can find more
 information at <https://thingpedia.stanford.edu/about>.
 
+## Installation
+
+This assumes that you have [node](https://github.com/nodejs/node) and [yarn](https://github.com/yarnpkg/yarn) installed, with at least **2 GB** of available diskspace. Please be sure to be on a **fast internet connection** before going through the installation.
+
+1. Clone this repo on your machine.
+```
+git clone https://github.com/stanford-oval/almond-server.git
+```
+
+2. Install libpulse.
+```
+# On Ubuntu/Debian
+sudo apt-get install libpulse-dev
+```
+
+3. Install [mimic](https://github.com/MycroftAI/mimic1).
+
+4. Install all packages and build bundle. 
+```
+yarn
+yarn build
+```
+
+5. Start Almond server.
+```
+yarn start
+```
+
+6. Navigate to [localhost:3000](http://localhost:3000).
+
+## Experimental Chat Interface
+
+While Almond Server already has a chat interface built using jQuery, the Almond team is currently working on an experimental chat interface written in React that you can try.
+
+### Setup
+
+To use this, first create an Almond developer account [here](https://almond.stanford.edu/user/register). Next, log in to your account and click on [Settings](https://almond.stanford.edu/user/profile) on the top right. Scroll down to "Authorized third-party apps" and click "Issue an Access Token". Copy and paste this access token into a `.env` file under the `REACT_APP_ACCESS_TOKEN` variable like this
+
+```
+REACT_APP_ACCESS_TOKEN="eyJhbGsomethingsomethingsomething"
+```
```

Comment:
![wat](https://ih1.redbubble.net/image.708418323.9485/pp,550x550.u2.jpg)

Why do you need a Cloud Almond access token, inside Almond Server?

Reply:
We don't need it yet (was just used for the chat video demo). We'll eventually need something, however, to regulate our TTS and STT API, no?

Reply:
Probably. But it will be an API key not an OAuth token.